lib/matplotlib/collections.py

```diff
@@ -2016,6 +2015,57 @@ def set_paths(self):
             self._meshWidth, self._meshHeight, self._coordinates)
         self.stale = True
 
+    def set_array(self, A):
+        if isinstance(self, QuadMesh) and A is not None:
```

Comment:
The isinstance check is not needed. The method is defined in QuadMesh and thus the check is true for all reasonable usages.

Reply:
Ah yes, sorry I shall make this change.